scripts/tools/zap_regen_all.py

```diff
@@ -65,6 +66,50 @@ class TargetType(Flag):
 }
 
 
+class ZapInput:
+    """ZAP may be run from a .zap configuration or from zcl properties.
+
+    Running from a '.zap' configuration will load the zcl properties from
+    the '.zap' file, however it will also load cluster enabling and settings.
```

Comment:
```suggestion
    Running from a '.zap' configuration will load the cluster XML and
    it will also load cluster enabling and settings defined in the '.zap' configuration.
```